src/FTPWindow.cpp

```diff
@@ -104,7 +104,10 @@ TFTPWindow::TFTPWindow(BRect frame, const char *name)
 	float minW, minH, maxW, maxH;
 	this->GetSizeLimits(&minW, &maxW, &minH, &maxH);
 	this->SetSizeLimits(280, maxW, 250, maxH);
-	
+
+	fOpenPanel.SetTarget(BMessenger(this));
+	fOpenPanel.SetButtonLabel(B_DEFAULT_BUTTON, B_TRANSLATE("Upload"));
+
```

Comment:
We should change the panel's window title as well:
fOpenPanel.Window()->SetTitle(B_TRANSLATE("FtpPositive: Upload"));

Reply:
Good point...